Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp

```diff
@@ -11,7 +11,7 @@ namespace Hazel {
 	{
 	}
 
-	void OrthographicCameraController::OnUpdate(Timestep ts)
+	void OrthographicCameraController::OnUpdate(const Timestep& ts)
```

Comment:
Why make these const ref?

Reply:
because it's not necessary to be copied ...

Reply:
Why does that matter In this case? Timestep contains a single float that is created on the stack, using this now means the ABI is passing around a pointer equivalent that it must dereference on usage, instead of just a struct decomposed to a float.

While these optimizations are insignificant and unnecessary based on this struct I think it will be the opposite of an optimizations, as it now means the caller needs to store it in an addressable location (not registers staying in registers as could be done) and then has the overhead of loading from the address on it's usage